Example/android/app/build.gradle

```diff
@@ -73,8 +73,8 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    entryFile: "index.js",
-    enableHermes:true,
+        entryFile   : "index.js",
```

Comment:
Hi @davinctor  great work!

seems to be unexpected changes no?

Reply:
Actually expected, usual autoformatter style for `.gradle` files.
Of course, If it would be a problem I will revert these changes.